src/components/v2v3/V2V3Project/ProjectDashboard/components/CyclesPayoutsPanel/hooks/useCurrentUpcomingSubPanel.ts

```diff
@@ -35,6 +35,12 @@ export const useCurrentUpcomingSubPanel = (type: 'current' | 'upcoming') => {
     return upcomingFundingCycle?.duration?.isZero() ?? true
   }, [fundingCycle?.duration, type, upcomingFundingCycle?.duration])
 
+  /** Determines if the CURRENT cycle is unlocked.
+   * This is used to check if the upcoming cycle can start at any time. */
+  const currentCycleUnlocked = useMemo(() => {
+    return fundingCycle?.duration?.isZero() ?? true
```

Comment:
**suggestion**: Is this equivalent?

```suggestion
    return Boolean(fundingCycle?.duration?.isZero())
```

Reply:
@aeolianeth no. If `fundingCycle?.duration?.isZero()` is null or undefined, the null coalescing operator returns `true`, where casting as a Boolean would return `false` (it is falsy).

Although I didn't look TOO deeply into this, I assume that if the funding cycle's duration were undefined, it means that there isn't a current cycle (maybe launched without one, for example). I didn't double check this, but it matches a pattern used in the other cycleUnlocked check.

Reply:
true fact, all g